bin/consensus-tests.rs

```diff
@@ -976,16 +979,33 @@ async fn main() -> anyhow::Result<()> {
         let e = entry.unwrap();
 
         if e.file_type().is_file() {
-            res += run_test_file(e.path(), &test_names, transaction_test).await;
+            let p = e.into_path();
+            tasks.push(tokio::spawn(async move {
+                return run_test_file(p.as_path(), transaction_test).await;
+            }));
         }
     }
 
+    for task in tasks {
+        res += task.await.unwrap();
+    }
+
     res.skipped += skipped;
-    println!("Ethereum Consensus Tests:\n{:?}", res);
+    println!(
+        "Ethereum Consensus Tests:\n{:?}\nElapsed {:?}",
+        res,
+        now.elapsed()
+    );
 
     if res.failed > 0 {
         std::process::exit(1);
     }
+}
 
-    Ok(())
+fn main() {
+    Builder::new_multi_thread()
+        .thread_stack_size(32 * 1024 * 1024)
+        .build()
+        .unwrap()
+        .block_on(run());
```

Comment:
```suggestion
    Builder::new_multi_thread()
        .enable_all()
        .thread_stack_size(32 * 1024 * 1024)
        .build()
        .unwrap()
        .block_on(run());
```

Just in case we add any timers, to not have any surprises.